documentation/DotNetCli.md

```diff
@@ -25,6 +30,18 @@ To supply a ruleset file and a stylecop.json configuration file to the compiler
 
 **Note: ** `additionalArguments` is not currently defined in the schema but does exist and is passed during the build
 
+On VS VS2017 / dotnet core 1.1.1 SDK 1.0.1 projects update `.csproj` as follows to apply settings and custom rules:
+```xml
+    <PropertyGroup>
+        ...
+        <CodeAnalysisRuleSet>stylecop.ruleset</CodeAnalysisRuleSet>
+    </PropertyGroup>
+    <ItemGroup>
+        <None Remove="stylecop.json" />
+        <AdditionalFiles Include="stylecop.json" />
```

Comment:
Yeah, I also had issues until I upgraded to the latest beta version of stylecop. However, I did end up with a successful migration to csproj on an existing codebase